doc/webui/README.md

```diff
@@ -117,10 +111,9 @@ API to allow instances to register themselves with `clusterd`
 
 + Request (application/json)
 
-        {
-            "name": "auto-registered instance",
-            "host": "192.168.0.8"
-        }
+    + Attributes (Base Instance)
+        + host: 192.168.0.8
+        + name: autoregistered instance
```

Comment:
I only tried overriding the description, so I cannot really give you a good answer on this. But why do you want to override the examples?